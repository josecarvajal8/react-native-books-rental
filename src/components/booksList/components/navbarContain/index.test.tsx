import React from 'react';
import renderer from 'react-test-renderer';
import NavBarContain from '.';
const mockProps = {
    searchValue: '',
    onChangeSearchValue: () => null
}
test('Render booksList component correctly', () => {
    const bookListCmp = renderer.create(<NavBarContain  {...mockProps} />).toJSON();
    expect(bookListCmp).toMatchSnapshot();
})