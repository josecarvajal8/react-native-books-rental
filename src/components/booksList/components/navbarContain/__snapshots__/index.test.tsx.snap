// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render booksList component correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginTop": 7.444196428571429,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderWidth": 1.8115942028985508,
        "height": 29.776785714285715,
        "marginLeft": 36.231884057971016,
        "opacity": 1,
        "width": 36.231884057971016,
      }
    }
  />
  <Text>
    LIBRARY
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderWidth": 1.8115942028985508,
        "height": 29.776785714285715,
        "marginRight": 36.231884057971016,
        "opacity": 1,
        "width": 36.231884057971016,
      }
    }
  />
</View>
`;
